src/runtime/audience-action-flow-test.js

```diff
@@ -64,14 +104,20 @@ describes.realWin('AudienceActionFlow', {}, (env) => {
       const messageLabel = messageType.label();
       messageMap[messageLabel] = cb;
     });
-    sandbox.stub(runtime, 'win').returns({
-      location: {href: WINDOW_LOCATION_DOMAIN + '/page/1'},
-      document: win.document,
-    });
+    sandbox.stub(runtime, 'win').returns(win);
     onCancelSpy = sandbox.spy();
     sandbox.useFakeTimers(CURRENT_TIME);
   });
 
+  function setGtagToNonfunction() {
+    runtime.win.restore();
+    sandbox.stub(runtime, 'win').returns(
+      Object.assign({}, win, {
```

Comment:
What about removing the property which is the expected condition when gtag is not available?
`delete win.gtag`